pimd/pim_cmd.c

```diff
@@ -6861,6 +6877,34 @@ DEFUN (no_ip_pim_packets,
 	return CMD_SUCCESS;
 }
 
+DEFPY (igmp_group_watermark,
+       igmp_group_watermark_cmd,
+       "ip igmp watermark-warn (10-60000)$limit",
+       IP_STR
+       IGMP_STR
+       "Configure group limit for watermark warning\n"
+       "Group count to generate watermark warning\n")
+{
+	PIM_DECLVAR_CONTEXT(vrf, pim);
+	pim->igmp_watermark_limit = limit;
+
+	return CMD_SUCCESS;
+}
+
+DEFPY (no_igmp_group_watermark,
+       no_igmp_group_watermark_cmd,
+       "no ip igmp watermark-warn",
```

Comment:
I think `[(10-60000)$limit]` should be covered as well

Reply:
'no' command just resets the value to 0. Giving a count option in no command may confuse user and needs additional check to check if the no is executed for configured value. I guess we can keep it simple to tell pimd that we dont need warning anymore for any limit.

Reply:
Sorry, this is incorrect, you need to add the ability to accept the value here. This is an FRR convention, we do this so you can copy-paste config and add `no` at the front to undo them.

Reply:
Done.